tpm2/encoding_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"testing"
 
 	"github.com/google/go-tpm/tpmutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
I'd like @josephlr or @twitchy-jsonp input on introducing this (and other) dependencies here

Reply:
:+1:

Reply:
Assert in Go is discouraged over simple if statements. Perhaps a bunch of the bits below can be refactored into:

```go

if got, want := thing.Output(), "wanted"; got != want {
  t.Errorf("thing.Output() = %v, want %v", got, want)
}
```

Reply:
Alright. Thank you :)